test/screens/play/play_page_test.dart

```diff
@@ -34,4 +41,39 @@ void main() {
     expect(find.byType(ScryingPage), findsNothing);
     expect(find.byType(PlayerHand), findsOneWidget);
   });
+
+  testWidgets('should change colour based on the card type displayed',
+      (WidgetTester tester) async {
+    const plunderAWreck = PlunderAWreck();
+    const rumoursOfAnIsland = RumoursOfAnIsland();
+    const mutiny = Mutiny();
+    final GameState state =
+        makeGameState(deck: [plunderAWreck, rumoursOfAnIsland, mutiny]);
+    state.nextCard();
```

Comment:
I think we could drop the `nextCard` stuff and test more in isolation, like:

```
await tester.pumpWidget(createWidgetForTesting(child: CardDisplay(makeGameState(), PlunderAWreck()));
final basicCard = tester.widget<Card>(cardFinder);
expect(basicCard.color, basicCardColour);

await tester.pumpWidget(createWidgetForTesting(child: CardDisplay(makeGameState(), RumoursOfAnIsland()));
final questCard= tester.widget<Card>(cardFinder);
expect(questCard.color, questCardColour);
```

Just cuts out some of the noise, and makes the test a bit more isolated from other bits of functionality that could change.